packages/graphics/src/Graphics.ts

```diff
@@ -303,20 +374,21 @@ export class Graphics extends Container
      * @param {boolean} [options.native=false] - If true the lines will be draw using LINES instead of TRIANGLE_STRIP
      * @return {PIXI.Graphics} This Graphics object. Good for chaining method calls
      */
-    lineTextureStyle(options)
+    public lineTextureStyle(options: ILineStyleOptions | number = null): Graphics
```

Comment:
we should remove number as a type for options since it’s deprecated.

Reply:
we can remove deprecated behavior because it is deprecated.

Reply:
No we should leave the if block because it still needs to work at runtime, but we should have docs or types show the old params.

Reply:
agree with @bigtimebuddy . This place needs an object. When we make new typings people will have to fix MANY things in their TS code, it will be like major release.

Reply:
Done